docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/ApplyOnnxModel.cs

```diff
@@ -6,10 +6,10 @@
 
 namespace Samples.Dynamic
 {
-    public static class OnnxTransformExample
+    public static class ApplyOnnxModel
     {
         /// <summary>
-        /// Example use of OnnxEstimator in an ML.NET pipeline
+        /// Example use of ApplyOnnxModel in an ML.NET pipeline
         /// </summary>
```

Comment:
let's delete this <summary> (it's stating the obvious and we don't have it for other samples)